fhir-operation-bulkdata/src/main/java/com/ibm/fhir/operation/bulkdata/client/BulkDataClient.java

```diff
@@ -128,6 +135,26 @@ private KeyStore loadKeyStoreFile(String ksFilename, String ksPassword) {
         }
     }
 
+    /**
+     * Generate a random AES key or 32 byte value encoded as a Base64 string.
+     *
+     * @return
+     */
+    private String getRandomKey() {
```

Comment:
Why not just use a UUID?

Reply:
I wanted to use simple UUID, but Lee thought UUID is not safe enough.